Makefile

```diff
@@ -52,9 +52,10 @@ offlinebundle:
 		. ./TEMPVENV/bin/activate; \
 		pip install -U pip setuptools; \
 		python setup.py sdist; \
-		pip download --dest=dist/offlinebundle/helper_packages --no-cache-dir  \
+		pip download --dest=dist/offlinebundle/helper_packages --no-cache-dir --only-binary :all: \
+						--implementation=py --abi=none --platform=any \
 						pip setuptools virtualenv virtualenvwrapper wheel appdirs \
-						pyparsing six packaging ; \
+						pyparsing six packaging pbr ; \
```

Comment:
The only reason I included it is because it was a dependency for a different package and pip complained when it couldn't find it.